webapp/platform/eslint-plugin/package.json

```diff
@@ -12,17 +12,23 @@
   "homepage": "https://github.com/mattermost/mattermost/tree/master/webapp/platform/eslint-plugin#readme",
   "main": "index.js",
   "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^7.1.0",
```

Comment:
Do we want these pinned? Or should the package try and defer to whatever the consumer uses?

Reply:
Generally, we should try to be looser with these. If we try to be exact, we're going to get people stuck in dependency hell as soon as they try to use our stuff. The real reason we use exact versions in other `package.json` files goes back to NPM not having lockfiles and then having crappy lockfiles for quite a while. I think they're solid now though